transparency-log.md

```diff
@@ -0,0 +1,51 @@
+# US-RSE Code of Conduct Transparency Log
+
+The US-RSE expects everyone who participates in the community will honor our
+[Code of Conduct](https://us-rse.org/about/code-of-conduct/). This includes
+anyone participating in any US-RSE-related activities, anyone claiming
+affiliation with the US-RSE, and especially anyone representing the US-RSE
+Association in any role (including as a volunteer or speaker).
+
+This log serves to inform the US-RSE community about all responses to code of
+conduct reports in a public, aggregated, and anonymized form. We believe that a
+community must be able to see that a Code of Conduct is enforced in order for it
+to be effective. Without a visible process potential community members may not
+feel comfortable joining. A visible log can also help the community establish
+the social standards expressed in the code of conduct.
+
+A public record allows the community to monitor the enforcement of the Code of
+Conduct and propose changes to the reporting and enforcement process if
+necessary.
+
+## Code of Conduct Reports and Responses
+
+Updated as of June 30, 2022
+
+### May 9, 2022
```

Comment:
I think the order of these three items should be reversed so that they are chronological.

Reply:
I have no preference on the order, I think either could work. I would go with whatever the CoC committee thinks is best.

Reply:
I'd vote for reverse chronological. Would it be worth a note?